electronics_lib/PriceGetter.py

```diff
@@ -25,47 +26,58 @@ def run(self) -> Dict[str, int]:
 
 class GeneratePrice(BaseBackend):
     # part_number -> [ [lower-bound_1, price_1], [lower-bound_2 (and the previous upperbound), price_2], ... ]
-    PRICE_TABLE = Dict[str, List[Tuple[int, float]]] = {}
+    PRICE_TABLE: Dict[str, List[Tuple[int, float]]] = {}
 
     @classmethod
     def get_price_table(cls) -> Dict[str, List[Tuple[int, float]]]:
         if not GeneratePrice.PRICE_TABLE:
-            cur_path = os.path.dirname(__file__)
-            parts_library = os.path.relpath('resources/Pruned_JLCPCB SMT Parts Library(20220419).csv', cur_path)
+            current_path = os.getcwd()
+            parts_library = current_path + "\PolymorphicBlocks\electronics_lib\\resources\Pruned_JLCPCB SMT Parts Library(20220419).csv"
```

Comment:
I'd recommend using the `PartsTable.with_source_dir` utility function like in [JlcPart](https://github.com/BerkeleyHCI/PolymorphicBlocks/blob/master/electronics_lib/JlcPart.py#L48) so the path is relative to this file. `os.getcwd` will probably differ based on whether it's within the IDE or run from the command line as eg blinky_skeleton.py or unit tests.